Source/Private/_ASPendingState.mm

```diff
@@ -586,9 +595,26 @@ - (NSString *)accessibilityLabel
 
 - (void)setAccessibilityLabel:(NSString *)newAccessibilityLabel
 {
-  _flags.setAccessibilityLabel = YES;
   if (accessibilityLabel != newAccessibilityLabel) {
```

Comment:
Hm, are we comparing strings here?

Reply:
Yes, but this is from the original implementation: https://github.com/fruitcoder/Texture/blob/master/Source/Private/_ASPendingState.mm#L590

Reply:
Yes, I see you follow this implementation throughout your changes. I should have asked better questions, sorry. 

Is there any reason to compare memory addresses here? Is the original implementation correct/acceptable?

Reply:
TBO I would compare those Strings with `isEqualToString:`, since if they are equal (by means of isEqualToString), we don't have to do anything.

Reply:
I agree. I'd do it the right way and not concern about the perf of `isEqualToString`, compared to `==`, especially because those labels aren't changed often I believe. Would you mind take this opportunity to do an audit in this file and switch to the formally correct compare methods?

Reply:
The existing code makes some sense, since a common use case for accessibility is to use constants for them.

The pointer compare made more sense when (1) CPUs were way slower and (2) this was an in-house framework.

I agree we should change to use `isEqualToString:`. 

Also just below here, we don't need to explicitly copy the newAccessibilityLabel before passing it to `initWithString:`. That initializer copies the contents ("initialized with the characters of `aString`").